### DISCUSSION TITLE
PHP Intl extension

### DISCUSSION THREAD

**Discussion by maciejfijalkowski** [symfony/symfony#45613] [Q&A] [answered]
Hi, I have a problem.
Im using Symfony 5 and EasyAdmin
Im working on Windows and on localhost I start server by just typing: symfony server:start
When I enter entity page on EasyAdmin, when entity use TimestampableEntity I have error: 

> When using date/time fields in EasyAdmin backends, you must install and enable the PHP Intl extension, which is used to format date/time values.

How to fix this?

**Comment by lyrixx** [comment]
You need to install / enable the intl extension.
To to that you need to refer to the documation of you PHP provider.

For example, if you used wamp, and IIRC you can right click on the icon > extension > intl
If you use wsl with ubuntu/debian : something like `sudo apt install php-intl`

**Comment by helmis123** [comment, marked as answer]
On windows just remove semicolon in php.ini
`;extension=gmp`
`extension=intl`
And restart server.

**Reply by maciejfijalkowski** [reply]
Its worked

**Reply by laurinaT** [reply]
Saved my life... Thanks <3

**Reply by ghost** [reply]
Thannnkssss

**Reply by DaweLionel** [reply]
merci

**Reply by Hasiniaina007** [reply]
thanks

**Reply by kimon-vasileiadis** [reply]
Didn't work

**Comment by josfaber** [comment]
I had to do two things on my Symfony 6.1 project to make this work:
1. change Dockerfile to install Intl extension (although is was already in there)
2. install symfony/intl via docker after rebuild of image

Dockerfile changes:
FROM
```
# persistent / runtime deps
RUN apk add --no-cache \
		acl \
		fcgi \
		file \
		gettext \
		git \
	;

RUN set -eux; \
    install-php-extensions \
    	intl \
    	zip \
    	apcu \
	opcache \
    ;
```
TO
```
# persistent / runtime deps
RUN apk add --no-cache \
		acl \
		fcgi \
		file \
		gettext \
		git \
		icu-dev \
	;

RUN set -eux; \
    install-php-extensions \
    	zip \
    	apcu \
	opcache \
    ;

RUN docker-php-ext-configure intl && \
	docker-php-ext-install intl \
	;
```

Then, after rebuild of the image, run this in the container: 
    `composer require symfony/intl`
